integration-tests/deployment/keystone/deploy_test.go

```diff
@@ -0,0 +1,181 @@
+package keystone_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/stretchr/testify/assert"
+	"github.com/test-go/testify/require"
+
+	chainsel "github.com/smartcontractkit/chain-selectors"
+
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment"
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment/clo"
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment/clo/models"
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment/keystone"
+	kcr "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/capabilities_registry"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+func TestDeploy(t *testing.T) {
+	lggr := logger.TestLogger(t)
+
+	t.Run("memory chains clo offchain", func(t *testing.T) {
```

Comment:
why the single subtest here vs just top level?

Reply:
fixed; left over cruft